src/vivarium_gates_mncnh/components/antenatal_care.py

```diff
@@ -0,0 +1,291 @@
+from __future__ import annotations
+
+from typing import Callable
+
+import numpy as np
+import pandas as pd
+import scipy.stats as stats
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+from vivarium.framework.population import SimulantData
+from vivarium.framework.state_machine import Machine, State, Transient, Transition
+from vivarium.types import ClockTime
+
+from vivarium_gates_mncnh.constants.data_values import (
+    ANC_RATES,
+    COLUMNS,
+    LOW_BIRTH_WEIGHT_THRESHOLD,
+    SIMULATION_EVENT_NAMES,
+    ULTRASOUND_TYPES,
+)
+from vivarium_gates_mncnh.utilities import get_location
+
+
+class TreeMachine(Machine):
```

Comment:
When I do a refactor Ill move these general clases into a separate module but for now I want it consolidated into one file